src/Core/AbstractStack.php

```diff
@@ -19,11 +19,17 @@ abstract class AbstractStack
      */
     public $stackOptions;
 
-    public function __construct($name = null, array $stackOperations = [], array $stackOptions = [])
+    /**
+     * @var array Array of stack variables that this stack has
```

Comment:
I assume a stack variable is a string? Can you specify the array type please? :)

Reply:
scalar[] it  is.

Reply:
or rather `array<bool|float|int|string>`